youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -379,6 +380,8 @@ def run(self, information):
             # Don't copy the existing subtitles, we may be running the
             # postprocessor a second time
             '-map', '-0:s',
+            # Don't copy Apple TV subtitles format, bin_data
```

Comment:
It's a chapters track as explained in the ffmpeg issue you linked that appears after chapters metadata is embedded by `--add-metadata`.